.github/ct-install-config/chart_schema.yaml

```diff
@@ -17,7 +16,8 @@ condition: str(required=False)
 keywords: list(str(), required=False)
 tags: str(required=False)
 maintainers: list(include('maintainer'))
-annotations: map(str(), str(), required=False)
+annotations:
+  title: str(required=True)
```

Comment:
> As of [v3.3.2](https://github.com/helm/helm/releases/tag/v3.3.2), additional fields are not allowed. The recommended > approach is to add custom metadata in annotations.

https://helm.sh/docs/topics/charts/

Reply:
Key `title` also works, but using the recommended approach I think it might be safer in the long run?
In case Helm decide to hard fail if unexpected keys found?